frontend/cypress/integration/common/graph_display-pf.ts

```diff
@@ -497,6 +497,8 @@ When('user {string} {string} traffic option', (action: string, option: string) =
 
 Then('{int} edges appear in the graph', (graphEdges: number) => {
   cy.waitForReact();
+  ensureKialiFinishedLoading();
```

Comment:
Is this necessary as this is also in:

https://github.com/kiali/kiali/blob/3e9b2a2588384e1bd371696834fc3fa02736c055/frontend/cypress/integration/common/graph_display-pf.ts#L33

Reply:
I don't think this command is necessary because it already waits for React to finish. However, it doesn't cause any harm either.